ios/PurchasesHybridCommon/PurchasesHybridCommonSwift/PurchasesExtensions.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  PurchasesExtensions.swift
+//  PurchasesHybridCommonSwift
+//
+//  Created by Andrés Boedo on 4/13/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+// todo: remove testable
+@testable import Purchases.Purchases
+import PurchasesCoreSwift
+
+@objc public extension Purchases {
+
+    @objc(configureWithAPIKey:appUserID:observerMode:userDefaultsSuiteName:
+            platformFlavor:platformFlavorVersion:dangerousSettings:)
+        static func configure(apiKey: String,
+                              appUserID: String?,
+                              observerMode: Bool,
+                              userDefaultsSuiteName: String?,
+                              platformFlavor: String?,
+                              platformFlavorVersion: String?,
+                              dangerousSettings: DangerousSettings?) -> Purchases {
+        var userDefaults: UserDefaults?
+        if let userDefaultsSuiteName = userDefaultsSuiteName {
+            userDefaults = UserDefaults(suiteName: userDefaultsSuiteName)
+            guard userDefaults != nil else {
+                fatalError("Could not create an instance of UserDefaults with suite name \(userDefaultsSuiteName)")
+            }
+        }
+
+        return self.configure(withAPIKey: apiKey,
+                              appUserID: appUserID,
+                              observerMode: observerMode,
+                              userDefaults: userDefaults,
+                              dangerousSettings: dangerousSettings)
+    }
+
+
+    // todo: make internal
+    @objc static func setDefaultInstance(_ instance: Purchases) {
```

Comment:
where is this coming from?

Reply:
this will go away as soon as we migrate to 4.0.0. Currently, the implementation for this lives in v3, in a .h protected file. Which is less than great. 
For now I just have this for tests to compile, but once we move to 4.0.0 we'll be able to have a proper protected extension to Purchases to provide this functionality